Automattic/jetpack
# [Jetpack AI] Disclaimer

This adds a disclaimer to make it clear if you can trust Jetpack AI.
![Zrzut ekranu 2023-01-26 o 19 15 18](https://user-images.githubusercontent.com/3775068/214917447-9fcbfc65-4f65-457b-b4de-f91a2c0d1e7f.png)


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Use any of the AI blocks (search for `magic`)
* Click on the sidebar
* See disclaimer
